cluster/gce/config-test.sh

```diff
@@ -132,3 +132,5 @@ OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+# Overlay network settings
+OVERLAY_NETWORK=${OVERLAY_NETWORK:-true}
```

Comment:
Sorry, I was unclear.  No, GCE should not default to flannel, but it should
be a one-line change to enable.

On Tue, Nov 24, 2015 at 1:23 PM, Prashanth B <notifications@github.com>
wrote:

> In cluster/gce/config-test.sh
> <https://github.com/kubernetes/kubernetes/pull/17668#discussion_r45796693>
> :
>
> > @@ -132,3 +132,5 @@ OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
> >
> >  # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
> >  E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
> > +# Overlay network settings
> > +OVERLAY_NETWORK=${OVERLAY_NETWORK:-true}
>
> Yeah that's what I'm doing with the coming pr. I'm attaching to network
> provider, but translating to a kubelet flag called
> --experimental-flannel-overlay to differentiate it from the other network
> providers that aren't tightly integrated yet. Re: default to flannel, so
> kube-up produces a flannel cluster?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17668/files#r45796693>.
>